src/proposal.rs

```diff
@@ -0,0 +1,97 @@
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+use crate::state::{Config, Migration};
+use crate::ContractError;
+use cosmwasm_std::{to_binary, BalanceResponse, Deps, Env, Uint128, WasmQuery};
+use cw20::Cw20QueryMsg;
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+pub enum Proposal {
+    LotteryEveryBlockTime { block_time: Uint128 },
+    HolderFeePercentage { percentage: Uint128 },
+    DrandWorkerFeePercentage { percentage: Uint128 },
+    PrizesPerRanks { ranks: Vec<u64> },
+    JackpotRewardPercentage { percentage: Uint128 },
+    AmountToRegister { percentage: Uint128 },
+    SecurityMigration { migration: Migration },
+    DaoFunding { amount: Uint128 },
+    StakingContractMigration {},
+    PollSurvey,
+    // test purpose
+    NotExist,
+}
+
+const HOLDERS_MAX_REWARD: u8 = 20;
```

Comment:
```suggestion
const HOLDERS_MAX_REWARD: u8 = 100;
```
We should remove max limit 
Fix #12